docs/BR.md

```diff
@@ -1739,142 +1737,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
-
-   * `certificatePolicies:policyQualifiers:policyQualifierId` (Optional)
-
-     `id-qt 1` [RFC5280].
-
-   * `certificatePolicies:policyQualifiers:qualifier:cPSuri` (Optional)
-
-     HTTP URL for the Root CA's Certificate Policies, Certification Practice Statement, Relying Party Agreement, or other pointer to online policy information provided by the CA.
-
-b. `cRLDistributionPoints`
-
-   This extension MUST be present and MUST NOT be marked critical. It MUST contain the HTTP URL of the CA's CRL service.
-
-c. `authorityInformationAccess`
-
-   This extension SHOULD be present. It MUST NOT be marked critical.
-
-   It SHOULD contain the HTTP URL of the Issuing CA's certificate (`accessMethod` = 1.3.6.1.5.5.7.48.2).
-   It MAY contain the HTTP URL of the Issuing CA's OCSP responder (`accessMethod` = 1.3.6.1.5.5.7.48.1).
-
-d. `basicConstraints`
-
-   This extension MUST be present and MUST be marked critical. The `cA` field MUST be set true. The `pathLenConstraint` field MAY be present.
+### 7.1.2 Certificate Content and Extensions
+
+If the CA asserts compliance with these Baseline Requirements, all certificates that it issues MUST comply with one of the following certificate profiles:
+
+  * CA Certificates
+    * [Section 7.1.2.1.- Root CA Certificate Profile](#7121-root-ca-certificate-profile)
+    * Subordinate CA Certificates
+      * Cross Certificates
+        * [Section 7.1.2.2 - Cross-Certified Subordinate CA Certificate Profile](#7122-cross-certified-subordinate-ca-certificate-profile)
+      * Technically Constrained CA Certificates
+        * [Section 7.1.2.3 - Technically-Constrained Non-TLS Subordinate CA Certificate Profile](#7123-technically-constrained-non-tls-subordinate-ca-certificate-profile)
+        * [Section 7.1.2.4 - Technically-Constrained TLS Subordinate CA Certificate Profile](#7124-technically-constrained-tls-subordinate-ca-certificate-profile)
+      * [Section 7.1.2.5 - TLS Subordinate CA Certificate Profile](#7125-tls-subordinate-ca-certificate-profile)
+  * [Section 7.1.2.6 - Subscriber (End-Entity) Certificate Profile](#7126-subscriber-server-certificate-profile)
+    * Domain Validated
+    * Individual Validated
+    * Organization Validated
+    * Extended Validation
+  * __**TBD: Infrastructure Certificates**__
+    * [Section 7.1.2.7 - OCSP Responder Certificate Profile](#7127-ocsp-responder-certificate-profile)
+    * __**TBD: Infrastructure Cert**__
+    * __**TBD: Precert Signing CA**__
+    * __**TBD: Interoperability Testing Certs (valid/revoked/expired)?**__
+
+#### 7.1.2.1 Root CA Certificate Profile
+
+| __Field__                      | __Description__ |
+| ---                            | ------          |
+| `tbsCertificate`               | |
+| \ \ \ \ `version`              | MUST be v3(2) |
+| \ \ \ \ `serialNumber`         | MUST be a non-sequential number greater than zero (0) containing at least 64 bits of output from a CSPRNG. |
```

Comment:
I think additional clarification would be beneficial here to specify that the encoded value MUST be no more than 20 octets. This point isn't clear from 5280, but client implementations and precedent dictate this (https://github.com/zmap/zlint/issues/502).

Reply:
In line with your [other comment](https://github.com/sleevi/cabforum-docs/pull/36#discussion_r608869714), I instead called this Appendix B explicitly in 7.1.2.5, where this MUST NOT is already expressed, namely:

> Conforming CAs MUST NOT use serialNumber values longer than 20 octets.

I feel like that's pretty clear? :)

Reply:
I don't think I agree that provides sufficient clarity, as it is still not entirely clear whether the 20 octet limit is on the integer value or the DER encoding thereof. This actually came up on PKIX just a few days ago: https://mailarchive.ietf.org/arch/msg/pkix/5G0Nq3GBnwiMDCYshSYzJppGSHE/. I think having one-sentence guidance that "DER-encoded serial number values MUST be no longer than 20 octets" would be useful and help to avoid future incidents.

Reply:
I'm not entirely swayed here.

Beyond the fact that this has been discussed at great length, and *any* CA, whether new or existing, absolutely should be familiar with this as a best competency test to becoming a CA, I worry that this argument is effectively cargo-culting at this point. As the thread you link notes, for certificate serial numbers, it is explicit (the question was re: CRL numbers, which lacks such explicitness).

I understand fundamentally the question is "Is asking folks to say 'shibboleth' fair? Can't we pick a word easier to pronounce", and I think that does sort of miss the point of the word in the first place. But sure, we can go ahead and make it easier for folks to ignore reading RFC 5280 by adding this, at the continued risk of more misunderstandings or "We didn't realize we had to read the BRs **and** RFC 5280", which regrettably continues to be a perception from CAs.